docs/resource-policy/quick-start.md

```diff
@@ -0,0 +1,75 @@
+# Quick-start
+
+The following describes the minimum number of steps to get started with NRI
+Resource Policy plugin.
+
+## Pre-requisites
+
+- containerd or cri-o container runtime installed and running, and also
+  NRI feature enabled.
+- kubelet installed on your nodes
+
+## Setup NRI Resource Policy Plugin
+
+First, compile the resource plugins and create deployment image.
+
+```
+make && make images
+```
+
+### Deploy Daemonset
+
+The build/ directory will contain the needed images and deployment
+files. Copy the plugin .yaml file and corresponding image file into
+the node and deploy it there.
+
+For example:
+
+```
+ls build/images
+   nri-resource-policy-balloons-deployment.yaml
+   nri-resource-policy-balloons-image-ed6fffe77071.tar
+   nri-resource-policy-topology-aware-deployment.yaml
+   nri-resource-policy-topology-aware-image-9797e8de7107.tar
+```
+
+Copy the nri-resource-policy-topology-aware-deployment.yaml and the
+latest tar file, that was generated by the `make images` command.
```

Comment:
Also here I'd suggest to instruct the user to push to a container registry and use that instead of manually copying images around and loading to nodes